src/communication/dryrun_global_policy.hpp

```diff
@@ -74,6 +74,13 @@ struct dryrun_global_policy {
         return size()*value;
     }
 
+    template <typename T>
```

Comment:
SHould this also be added to the Global_policy interface.
It also might need some api description/comment